Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerViewController.swift

```diff
@@ -279,16 +282,34 @@ open class MessageComposerViewController<ExtraData: ExtraDataTypes>: ViewControl
     }
     
     // MARK: Suggestions
-    
-    func showSuggestionsViewController(
-        with state: SuggestionsViewControllerState,
-        onSelectItem: ((Int) -> Void)
-    ) {
-        suggestionsPresenter?.showSuggestionsViewController(with: state, onSelectItem: onSelectItem)
+
+    public func showSuggestionsViewController(for kind: SuggestionKind, onSelectItem: @escaping ((Int) -> Void)) {
```

Comment:
What will happen if you call this while the suggestion VC is already presented. Shouldn't we handled that edge case?